scrypt_test.go

```diff
@@ -106,3 +107,18 @@ func TestDecodeHash(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateFromPasswordWithLimit(t *testing.T) {
```

Comment:
As above - should test with a range of inputs (0ms duration, 0MB memory; 500ms duration, 1MB memory; 50ms duration, 4MB memory).